src/pages/teachers/programmes/[programmeSlug]/units/[unitSlug]/lessons.tsx

```diff
@@ -203,15 +205,17 @@ export const getStaticProps: GetStaticProps<
         throw new Error("unexpected context.params");
       }
 
-      const curriculumData = isSlugLegacy(programmeSlug)
-        ? await curriculumApi.lessonListing({
-            programmeSlug,
-            unitSlug,
-          })
-        : await curriculumApi2023.lessonListing({
-            programmeSlug,
-            unitSlug,
-          });
+      const curriculumData =
+        isSlugLegacy(programmeSlug) &&
+        !programmeSlug.endsWith("early-years-foundation-stage-l")
```

Comment:
we're doing this in a bunch of places now so we should probably pull it into its own method like `isSlugLegacy` is (and maybe a single method that combines both as well?)

Reply:
New functions added ✅